### PR TITLE
W 4 health index

### PR DESCRIPTION
**Pull request by Francesdenaro**

### CODE REVIEW

File: src/app/app.js [hunk 1]
```diff
@@ -11,12 +11,14 @@ import configureStore from './store/configureStore';
...
 import suffix from './components/suffix';
 import tabs from './components/tabs';
+import healthIndex from './components/healthIndex';
 import Timer from './components/Timer';
 import PlusOne from './components/plus-one/PlusOne';
 
 const app = (config) => {
     init(cookieLaw, document.getElementById('cookie-law'));
     init(suffix, document.querySelector('.js-suffix'));
+    init(healthIndex, document.querySelector('.js-health-index'));
```
Comment (fidelman) [inline src/app/app.js:22]: 👍

File: src/styles/base/heading.scss [hunk 2]
```diff
@@ -1,4 +1,16 @@
+.h2 {
```
Comment (fidelman) [inline src/styles/base/heading.scss:1]: 👍

File: src/styles/components/health-index.scss [hunk 3]
```diff
@@ -0,0 +1,307 @@
+.health-index {
```
Comment (fidelman) [inline src/styles/components/health-index.scss:1]: 👍

File: src/styles/variables.scss [hunk 4]
```diff
@@ -4,13 +4,17 @@ $primary: #ff6900;
...
 $border-light: #eee;
 $light-box-bg: #f6f6f6;
 $dark-box-bg: #c6c6c6;
+$section-title: #4c5459;
+$health-icon: #7e7e7e;
 
 // Border 
 
 $smooth-border: 4px;
+$smooth: 4px;
```
Comment (fidelman) [inline src/styles/variables.scss]: 16 and 17 are the same?

File: src/app/components/healthIndex.js [hunk 5]
```diff
@@ -0,0 +1,24 @@
+export default function healthIndex (container) {
+    const pointer = container.querySelector('.js-pointer');
+    const value = Number(container.querySelector('.js-value').innerHTML);
```
Comment (fidelman) [inline src/app/components/healthIndex.js]: where you should keep any data for the component?

File: src/app/components/healthIndex.js [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+
+    function isVisible(element) {
```
Comment (fidelman) [inline src/app/components/healthIndex.js]: use arrow functions from es6

File: src/app/components/healthIndex.js [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+            rect.top >= 0 &&
+            rect.left >= 0 &&
+            rect.bottom <= (window.innerHeight || container.documentElement.clientHeight) &&
+            rect.right <= (window.innerWidth || container.documentElement.clientWidth)
+        );
+    }
+
+    function animation() {
+        if (isVisible(pointer)) {
+            pointer.style.transform = `translateY(50%) rotate(${(value / 10) * 18}deg)`;
```
Comment (fidelman) [inline src/app/components/healthIndex.js]: should styles be in JS?

File: src/app/components/healthIndex.js [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+        }
+    }
+
+    animation();
```
Comment (fidelman) [inline src/app/components/healthIndex.js]: it is a function, it does smth it has to be a verb. -> animate.
Animate what? Animate arrow

Comment (fidelman) [reply src/app/components/healthIndex.js]: or pointer

File: src/app/components/healthIndex.js [hunk 9]
```diff
@@ -0,0 +1,24 @@
+export default function healthIndex (container) {
```
Comment (fidelman) [inline src/app/components/healthIndex.js:3]: are you sure your eslint is clear?

**Review by fidelman — CHANGES_REQUESTED** [review]

File: src/app/components/healthIndex.js (hunk 6 above)
Comment (fidelman) [reply src/app/components/healthIndex.js]: recommend you to put it to helpers. maybe you will use it in the future

File: src/styles/variables.scss (hunk 4 above)
Comment (Francesdenaro) [reply src/styles/variables.scss]: smooth-border is used for the width, smooth for the radius

**Pull request merged**